app/controllers/notifications_controller.rb

```diff
@@ -45,7 +45,15 @@ def archive_selected
     head :ok
   end
 
-  def mark_as_read
+  def mark_read_selected
```

Comment:
I don't like that mark_read and mark_read_selected have different code paths and behave slightly differently (mark_read_selected updates GitHub and mark_read doesn't)